### DISCUSSION TITLE
2023-11-28 Meeting Agenda

### DISCUSSION THREAD

**Discussion by aflackey** [ucum-org/ucum#285] [Meetings]
### Date: 2023-11-28, 3:00 – 4:00 pm EST
**Chair(s): Marjorie Rallins**

|**TIME** |**AGENDA TOPICS** |**LEAD(S)** |
| ----------- | -----| -----|
| 3:00 | Welcome | Marjorie Rallins |
| 3:05 - 3:20 | Governance | Virginia Riehl|
| 3:20 - 3:30 | UCUM Releases | Steven Wagers, Tim Briscoe|
| 3:30 - 3:50 | Proposal Review/Open Action Items | Eza Hafeza, All |
|3:50 - 3:55 | Additional Topics | All
|3:55 - 4:00 | Other Open action Items, Topics for Next Meeting | All
| 4:00 | Adjourn | Marjorie Rallins |

***OPEN ACTION ITEMS***
|**ITEM** |**DATE IDENTIFIED**|**TARGET COMPLETION**| **LEAD** |
| ----------- | -----| -----| -----|
|[GitHub #180] Proposal k[UA]/l  -  Proposal k[UA]/l to requestor, advising current use of U in UCUM standard; invite requestor to follow and/or engage in the GitHub issue | 2022-10-03 | 2023-05-27 presentation; awaiting decision | Eza Hafeza |
|[GitHub #182] Chemiluminescence Units – **Request more info from the submitter:** context, relativeness to base units|2022-10-03| On hold |Eza Hafeza|
|GitHub [multiple unit requests](https://github.com/ucum-org/ucum/issues?q=is%3Aissue+is%3Aopen+turbidity) Turbidity – **Request more information from submitter**: context, relativeness to base units|2022-10-03| 2023-06-27 presentation, awaiting decision |Eza Hafeza|

***LINKS TO REFERENCE MATERIALS***
[2023 UCUM Units Proposed and Resolved_09232023 (Gsheet)](https://docs.google.com/spreadsheets/d/1x5v5h9KgUBPx75H0qjifh1YSz8Igz6z9/edit?usp=sharing&ouid=110875911742539584915&rtpof=true&sd=true)
[IgE references (folder)](https://drive.google.com/drive/folders/10E4AS8cGLYnh1vkrIhloUQUKjH9KafGW)
[Turbidity Concepts_William Smith_2023-06-27 (slide deck)](https://docs.google.com/presentation/d/16Us66SJhKc2sq0mEmNWzLx5wucTM0LD1/edit#slide=id.p1)
[Sadar - Making Sense of Turbidity (PDF)](https://drive.google.com/file/d/1Ryt6wyiUxR8dxY-vPJkcdRol6wV23ZyG/view?usp=drive_link)

@ucum-org/regenstrief 
@ucum-org/advisors

**Comment by ucumworker** [comment]
-- Please reply above this line --

Hello Ucum-Org/ucum,

We received your email request and will get back to you as soon as we
can. Please know our response time does vary at different times of the
year due to our bi-annual release. 

Thank you,

Best Regards,
The LOINC Team
